src/Symfony/Component/Security/Tests/Core/Encoder/BCryptPasswordEncoderTest.php

```diff
@@ -45,9 +45,6 @@ public function testCostInRange()
         }
     }
 
-    /**
-     * @requires PHP 5.3.7
```

Comment:
Not sure why these two tests were skipped. They work perfectly fine with all PHP versions, and they're not skipped on 2.7.